great_expectations/experimental/datasources/sources.py

```diff
@@ -28,15 +30,15 @@
         DataAsset,
         Datasource,
     )
+    from great_expectations.validator.validator import Validator
 
 SourceFactoryFn = Callable[..., "Datasource"]
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_PANDAS_DATASOURCE_NAMES: tuple[str, str] = (
-    "default_pandas_datasource",
-    "default_pandas_fluent_datasource",
-)
+DEFAULT_PANDAS_DATASOURCE_NAME: Final[str] = "default_pandas_datasource"
+
+DEFAULT_PANDAS_DATA_ASSET_NAME: Final[str] = "#ephemeral_data_asset"
```

Comment:
@NathanFarmer Quick Question: What is the meaning of the "#" in the name?  Thanks.

Reply:
The product request was that we give this ephemeral asset a `$` prefix to denote that it will not be serialized. The `$` does not work, because GX attempts to perform config variable substitution on any variable in the context that starts with `$`. I opted to use `#` instead and will update this if requested once we get product feedback.

Reply:
@alexsherstinsky

Reply:
🙇